documentation/source/user_section/command-line/sct_deepseg.rst

```diff
@@ -5,12 +5,38 @@ sct_deepseg
 
 Here we provide a gallery of each model available in the ``sct_deepseg`` CLI tool.
 
-Spinal cord
+Contrast agnostic approaches 
 -----------
```

Comment:
It looks like the documentation build is failing due to the following error:

```
/home/docs/checkouts/readthedocs.org/user_builds/spinalcordtoolbox/checkouts/4832/documentation/source/user_section/command-line/sct_deepseg.rst:9: WARNING: Title underline too short.
```

To fix this, we just need to extend the underline:

```suggestion
Contrast agnostic approaches
----------------------------
```